shodan/stream.py

```diff
@@ -27,9 +27,12 @@ def _create_stream(self, name, timeout=None):
         # If the user requested a timeout then we need to disable heartbeat messages
         # which are intended to keep stream connections alive even if there isn't any data
         # flowing through.
-        if timeout:
+        if timeout is not None:
             params['heartbeat'] = False
 
+        if query is not None:
```

Comment:
It feels a bit weird to handle this in ``_create_stream()`` instead of the ``custom()`` method as the ``query`` parameter is only used for custom streams. It might be nicer if we modify ``_create_stream()`` to have a ``**kwargs`` parameter whose key/ values get put into the ``params`` object.

Reply:
The downside of that is some kwargs would be treated differently than others then (ex. ``timeout`` wouldn't be put into the ``params`` object).